modules/worker/cloud-config.tf

```diff
@@ -216,6 +238,17 @@ write-files:
         copytruncate
       }
 
+  - path: /home/core/go-aws-mon.sh
+    permissions: '0755'
+    owner: core
+    content: |
+      #!/bin/bash -
+      if [ ! -x /home/core/go-aws-mon ]; then
+        wget -O /home/core/go-aws-mon https://github.com/a3linux/go-aws-mon/raw/aggregate/bin/go-aws-mon 
```

Comment:
@adambom  Shouldn't this be wget -O /home/core/go-aws-mon https://github.com/a3linux/go-aws-mon/raw/master/bin/go-aws-mon ? 

I get a 404 on https://github.com/a3linux/go-aws-mon/raw/aggregate/bin/go-aws-mon